commitizen/defaults.py

```diff
@@ -50,8 +50,8 @@ class Settings(TypedDict, total=False):
 
 name: str = "cz_conventional_commits"
 config_files: List[str] = [
-    "pyproject.toml",
     ".cz.toml",
+    "pyproject.toml",
```

Comment:
Wondering why we change the loading order

Reply:
You are right, it's not needed anymore